notebooks/s3d-mil-nce-text-to-video-retrieval/s3d-mil-nce-text-to-video-retrieval.ipynb

```diff
@@ -54,11 +54,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    
</pre><p>Why did you fix the version and not the range like tensorflow&gt;=2.5&lt;=2.16.1 ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Different versions of tensorflow, tensorflow-mac, tensorflow_hub and tf_kera have conflicts with each other. We want to ensure functionality in a certain environment. Changed package versions to the latest ones that can be combined.